packages/MarkdownEditor-Tests.package/MarkdownInlineParserTest.class/instance/testFindOpenerForEmphasisDelimiter.st

```diff
@@ -0,0 +1,30 @@
+tests - tokenization
+testFindOpenerForEmphasisDelimiter
+
+	| tokens opener closer foundOpener |
+	opener := MarkdownDelimiterEmphasis new 
+		content: '**'; 
+		leftFlanking: true; 
+		rightFlanking: false; 
+		yourself.
+	closer := MarkdownDelimiterEmphasis new 
+		content: '**'; 
+		leftFlanking: false; 
+		rightFlanking: true; 
+		yourself.
+		
+	tokens := OrderedCollection new.
+	tokens add: opener.
+	tokens add: (MarkdownString new content: 'f'; yourself).
+	tokens add: closer.
+	inlineParser tokens: tokens.
+	
+	inlineParser findOpenerFor: closer 
+		ifFound: [:o | foundOpener := o].
+	self assert: opener equals: foundOpener.
+	
+	tokens remove: opener.
+	inlineParser tokens: tokens.
+	inlineParser findOpenerFor: closer 
+		ifFound: [:o | self fail]
```

Comment:
Idea:
This is a separate test case. You modify the `tokens` collection so I had to keep the state mentally. I prefer to have one test with `{opener. string. closer}` and one with `{string. closer}`.
Also, when the first assert is moved in its own method, you can write `ifFound: [^ self assert: ...]. self fail`